modules/tensor_mechanics/test/tests/pressure/error.i

```diff
@@ -0,0 +1,99 @@
+[Mesh]
+  type = FileMesh
+  file = pressure_test.e
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Functions]
+  [./rampConstant]
+    type = PiecewiseLinear
+    x = '0. 1. 2.'
+    y = '0. 1. 1.'
+    scale_factor = 1.0
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+[]
+
+[AuxVariables]
+  [./pressure]
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
+  [../]
+[]
```

Comment:
You can use the tensor mechanics master action instead of the kernel and strain here, which ensures that we have consistency in the stress divergence use_displaced_mesh setting and the strain formulation

Reply:
I thought this was the action. Can you point me to an example what do you want me to use? I took this from `pressure_test.i`.

Reply:
This [test](https://github.com/idaholab/moose/blob/devel/modules/tensor_mechanics/test/tests/thermal_expansion/constant_expansion_coeff.i) is pretty close to what you want to use, and we have a small bit of documentation on the master action [here](http://www.mooseframework.org/moose/documentation/systems/Modules/TensorMechanics/Master/index.html) (more improvements coming soon!)

Turns out there is a much wider use of the legacy tensor mechanics action, that is restricted to just kernels, in our test repository than I'd like. Another issue for another day to make all of the tests use the master action.